src/Lykke.Quintessence.Api/ApiStartupBase.cs

```diff
@@ -0,0 +1,52 @@
+using System.Reflection;
+using FluentValidation.AspNetCore;
+using Lykke.Quintessence.Modules;
+using Lykke.Quintessence.Settings;
+using Lykke.Sdk;
+using Lykke.SettingsReader;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Lykke.Quintessence
+{
+    public abstract class ApiStartupBase<T> : ServiceStartupBase<AppSettings<T>>
+        where T : ApiSettings
+    {   
+        protected override bool IsDebug
+            => ApiStarter.IsDebug;
+
+        protected override string ServiceName
+            => "Api";
+
+        protected override void ConfigureFluentValidation(
+            FluentValidationMvcConfiguration configuration)
+        {
+            base.ConfigureFluentValidation(configuration);
```

Comment:
What is the desired flow to override validators?

Reply:
Let's leave it intact and refactor in future release